CONTRIBUTING.md

```diff
@@ -37,6 +40,21 @@ As a general rule of thumb, please try to create bug reports that are:
 - *Unique.* Do not duplicate existing tickets.
 - *Scoped to a Single Bug.* One bug per report.
 
+## Branching workflow
+MongoDB exporter uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html) `[major].[minor].[patch]`:
+- major version when you make incompatible changes
+- minor version when you add functionality in a backwards compatible manner
+- patch version when you make backwards compatible bug fixes
+
+`main` is a main branch where all the changes are merged **first**.
+
+`release-x.y` are the release branches and are forked from `main` to prepare and test the release. Release branch stays open to get critical patches and to release another _patch_ release. Release branch closes (no merges happens) just after new minor release is created.
+
+Each release is tagged with `vx.y.z` tag.
+
+
+Please submit your changes against `main` branch, if the fix is needed for the patch or minor release - please ask maintainers to cherry pick it into the release branch.
+
 ## Setup your local development environment
```

Comment:
```suggestion

```
We can remove those two and add them later once ready.

Reply:
didn't get it. Later in the chapter there is a desc on how to setup the env

Reply:
> didn't get it. Later in the chapter there is a desc on how to setup the env

Sorry, didn't notice there was a section. Pls ignore.